store/storetest/team_store.go

```diff
@@ -701,6 +787,36 @@ func testDelete(t *testing.T, ss store.Store) {
 	}
 }
 
+func testPublicTeamCount(t *testing.T, ss store.Store) {
+	o1 := model.Team{}
+	o1.DisplayName = "DisplayName"
+	o1.Name = "z-z-z" + model.NewId() + "b"
+	o1.Email = MakeEmail()
+	o1.Type = model.TEAM_OPEN
+	o1.AllowOpenInvite = true
+	_, err := ss.Team().Save(&o1)
+	require.Nil(t, err)
+
+	teamCount, err := ss.Team().AnalyticsPublicTeamCount()
```

Comment:
How about adding a private team too here so we are sure that isn't included in the count?

Reply:
I'll add this to the test.